sdk/core/azure-core/src/test/java/com/azure/core/implementation/logging/LoggingUtilsTests.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.logging;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class LoggingUtilsTests {
+    private static final String NEW_LINE = System.lineSeparator();
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = {"hello", "hello world\t123"})
+    public void noNewLine(String message) {
+        assertEquals(message, LoggingUtils.sanitizeLogMessageInput(message));
+    }
+
+    @ParameterizedTest
+    @MethodSource("messagesWithNewLine")
+    public void newLine(String message) {
+        assertEquals("hello", LoggingUtils.sanitizeLogMessageInput(message));
+    }
+
+    @Test
+    public void newLineOnly() {
+        assertEquals("", LoggingUtils.sanitizeLogMessageInput(NEW_LINE + NEW_LINE + NEW_LINE));
+    }
+
+    @Test
+    public void withCL() {
+        assertEquals("\thelloworld", LoggingUtils.sanitizeLogMessageInput("\r\thello\r" + NEW_LINE + "world"));
```

Comment:
I wonder if the expected value should be "t\hello world". i.e. should we replace \n or \r or \r\n with white space?